<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Anu" failures="2" tests="8" name="Test" time="4.478" errors="0" timestamp="2022-07-25T18:01:33 GMT-06:00">
  <testcase classname="Tests.GETTest01" name="getUser_Test1" time="1.568"/>
  <testcase classname="Tests.GETTest01" name="getUser_Test2" time="0.86"/>
  <testcase classname="Tests.GETTest01" name="getUser_Test4" time="0.281"/>
  <testcase classname="Tests.GETTest01" name="getUser_Test5" time="0.254"/>
  <testcase classname="Tests.GETTest01" name="getUser_invalidTest1" time="0.201">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
JSON path data.id[0] doesn&amp;apos;t match.
Expected: &amp;lt;4&amp;gt;
  Actual: &amp;lt;7&amp;gt;
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
JSON path data.id[0] doesn't match.
Expected: <4>
  Actual: <7>

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:86)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:58)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:270)
at io.restassured.specification.ResponseSpecification$body$1.callCurrent(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:50)
at io.restassured.specification.ResponseSpecification$body$1.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:117)
at io.restassured.internal.ValidatableResponseOptionsImpl.body(ValidatableResponseOptionsImpl.java:244)
at Tests.GETTest01.getUser_invalidTest1(GETTest01.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- getUser_invalidTest1 -->
  <testcase classname="Tests.GETTest01" name="getUser_invalidTest2" time="0.193">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
JSON path data.first_name doesn&amp;apos;t match.
Expected: (a collection containing &amp;quot;Michael&amp;quot; and a collection containing &amp;quot;George&amp;quot; and a collection containing &amp;quot;Rache&amp;quot;)
  Actual: &amp;lt;[Michael, Lindsay, Tobias, Byron, George, Rachel]&amp;gt;
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
JSON path data.first_name doesn't match.
Expected: (a collection containing "Michael" and a collection containing "George" and a collection containing "Rache")
  Actual: <[Michael, Lindsay, Tobias, Byron, George, Rachel]>

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:86)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:270)
at io.restassured.specification.ResponseSpecification$body$1.callCurrent(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:50)
at io.restassured.specification.ResponseSpecification$body$1.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:117)
at io.restassured.internal.ValidatableResponseOptionsImpl.body(ValidatableResponseOptionsImpl.java:244)
at Tests.GETTest01.getUser_invalidTest2(GETTest01.java:80)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- getUser_invalidTest2 -->
  <testcase classname="Tests.POSTTest" name="postUser_Test1" time="0.478"/>
  <testcase classname="Tests.PUTTest" name="putUser_Test1" time="0.542"/>
</testsuite> <!-- Test -->
